ros/lib/utils.py

```diff
@@ -31,6 +31,21 @@ def get_or_create(session, model, keys, **kwargs):
     return instance
 
 
+def delete_record(session, model, keys, **kwargs):
+    if not keys:
+        keys = kwargs.keys()
+    if isinstance(keys, str):
+        keys = [keys]
+    if not isinstance(keys, list):
+        raise TypeError('keys argument must be a string')
+    instance = session.query(model).filter_by(**{k: kwargs[k] for k in keys}).first()
+    deleted = instance.inventory_id
```

Comment:
Here you assumed that every instance of any model have `inventory_id` column. Is this correct?

Reply:
Yeah. Are there any cases where we won't have inventory_id for a system?

Reply:
Now, that we are returning the instance itself, we need not worry for the presence/absence of `inventory_id`.